pkg/model/stack.go

```diff
@@ -853,3 +853,13 @@ func (s composeServices) toGraph() graph {
 	}
 	return g
 }
+
+func (stack *Stack) GetServicesWithBuildSection() map[string]bool {
+	result := make(map[string]bool)
+	for name, service := range stack.Services {
+		if service != nil && service.Build != nil {
```

Comment:
@jLopezbarb This is not a robust way of checking which services have a build section, because it could be an empty struct. Do you know another way to check if its not _empty_

Reply:
I don't understand what you mean it could be an empty struct. When loading the file we create the struct so it can't ever be an empty struct.

Service can't ever be nil, and we should also check if we have to build from volume mounts(we create images by injecting the local path, otherwise the directory will be empty).

```suggestion
		if svc.Build != nil || len(svc.VolumeMounts) != 0 {
```